include/imex/Dialect/Dist/IR/DistOps.td

```diff
@@ -46,48 +75,149 @@ def Dist_Dialect : Dialect {
 class Dist_Op<string mnemonic, list<Trait> traits = []> :
     Op<Dist_Dialect, mnemonic, traits>;
 
-// Add function prototypes used for calling into distributed runtime
 def RuntimePrototypesOp : Dist_Op<"runtime_prototypes"> {
+    let summary = "Add function prototypes used for calling into distributed runtime";
 }
 
-// Register a ptensor of given shape with a (potentially distributed) runtime.
-// Returns an id to uniquely identify the tensor instance in future interactino with the runtime.
-// The runtime does not own or manage any PTensor memory. When needed by an operation,
-// (local) data needs to be provided.
-def RegisterPTensorOp : Dist_Op<"register_ptensor", []> {
```

Comment:
Why is `RegisterPTensorOP` removed?

Reply:
The new dialect no longer relies on a runtime to compute offsets, local shapes etc. PAsses generate code to compute in inline. This op got replace by `InitDistTensorOp`.